client_test.go

```diff
@@ -36,7 +31,7 @@ func ExampleHAProxyClient_Info() {
 	}
 	fmt.Printf("%s version %s\n", info.Name, info.Version)
 	// Output:
-	//HAProxy version 1.6.3
+	//HAProxy version 2.8.9-1ppa1~jammy
```

Comment:
hardcoding the version is that a good idea ? Or is better to hardcode the installed version too ? Or just do some kind of regex on that ?